deploy/olm-catalog/service-binding-operator/0.0.23/service-binding-operator.v0.0.23.clusterserviceversion.yaml

```diff
@@ -2,28 +2,59 @@ apiVersion: operators.coreos.com/v1alpha1
 kind: ClusterServiceVersion
 metadata:
   annotations:
-    alm-examples: '[{"apiVersion":"apps.openshift.io/v1alpha1","kind":"ServiceBindingRequest","metadata":{"name":"binding-request","namespace":"service-binding-demo"},"spec":{"applicationSelector":{"matchLabels":{"connects-to":"postgres","environment":"demo"},"group":"apps.openshift.io","version":"v1","resource":"deploymentconfigs"},"backingServiceSelector":{"group":"postgresql.baiju.dev","version":"v1alpha1","kind":"Database","resourceRef":"db-demo"}}}]'
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "apps.openshift.io/v1alpha1",
+          "kind": "ServiceBindingRequest",
+          "metadata": {
+            "name": "example-servicebindingrequest"
+          },
+          "spec": {
+            "applicationSelector": {
+              "group": "apps",
+              "resource": "deployments",
+              "resourceRef": "nodejs-rest-http-crud",
+              "version": "v1"
+            },
+            "backingServiceSelector": {
+              "group": "postgresql.example.dev",
+              "kind": "Database",
+              "resourceRef": "pg-instance",
+              "version": "v1alpha1"
+            },
+            "mountPathPrefix": "/var/credentials"
+          }
+        }
+      ]
     capabilities: Basic Install
-    containerImage: quay.io/redhat-developer/app-binding-operator:v0.0.23
-    description: An operator to support binding capabilities between imported apps
-      and operator backed services
+    categories: Developer Tools, OpenShift Optional, Integration & Delivery
+    containerImage: REPLACE_IMAGE
+    description: An operator to support binding capabilities between imported apps and operator backed services
   name: service-binding-operator.v0.0.23
   namespace: placeholder
 spec:
   apiservicedefinitions: {}
   customresourcedefinitions:
     owned:
-    - description: Expresses intent to bind an operator-backed service with a Deployment.
-      displayName: ServiceBindingRequest
```

Comment:
Yes, it should because now we have an additional annotation:

https://github.com/pedjak/service-binding-operator/blob/1a6a66f0327307454fc6f2f82bce5c0052dd8127/pkg/apis/apps/v1alpha1/servicebindingrequest_types.go#L76

and with that `displayName` is generated each time. We should treat `service-binding-operator.v0.0.23.clusterserviceversion.yaml ` more as base/template - all customization (additional fields/values) should be managed there.